src/lib/OpenEXRCore/string.c

```diff
@@ -120,10 +120,17 @@ exr_attr_string_create_with_length (
         /* someone might pass in a string shorter than requested length (or longer) */
         if (len > 0)
         {
+#ifdef _MSC_VER
```

Comment:
In my warnings cleanup elsewhere in OpenEXR, I'm just suppressing 4996 for the entire file. I can't think of a single case where spot-applying 4996 might be helpful. The warnings are just a constant reminder that Microsoft had an idea that never took hold in the programming community, outside of the windows centric community, anyway.